.github/workflows/release.yml

```
@@ -0,0 +1,27 @@
+on:
+  release:
+    types:
+      - published
+
+name: release
+
+jobs:
+  pypi:
+    name: upload release to PyPI
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: actions/setup-python@v2
+
+    - name: deps
+      run: python -m pip install -U setuptools build wheel
```

```suggestion
      run: python -m pip install -U build
```

These two should be unnecessary in the local environment if we're doing PEP 517 isolated builds with `build`.